payloads/configure.go

```diff
@@ -16,25 +16,29 @@
 
 package payloads
 
-// ServiceType is reserved for future use.
+// ServiceType is used to define the type of service from the
+// configurations.
 type ServiceType string
 
-// StorageType is reserved for future use.
+// StorageType is used to define the type of storage from the
+// configurations.
 type StorageType string
 
 const (
-	// Glance is reserved for future use.
+	// Glance is used to define the imaging service.
 	Glance ServiceType = "glance"
 
-	// Keystone is reserved for future use.
+	// Keystone is used to define the identity service.
 	Keystone ServiceType = "keystone"
 )
 
 const (
-	// Filesystem is reserved for future use.
+	// Filesystem is used to define storage type to use for the
```

Comment:
`Filesystem defines the local filesystem backend storage type for the configuration data`